bin/strata-client/src/errors.rs

```diff
@@ -10,6 +10,9 @@ pub enum InitError {
     #[error("missing init client state")]
     MissingInitClientState,
 
+    #[error("rollup name too long")]
```

Comment:
nit: The error message is not coherent with the enum variant. `InvalidRollupName` sounds like can occur even when the name is not long but does not match the expected one.